targettests/execution/bug_qubit.cpp

```diff
@@ -10,11 +10,13 @@
 
 // clang-format off
 // RUN: nvq++ %cpp_std --target anyon                    --emulate %s -o %t && %t | FileCheck %s
-// RUN: nvq++ %cpp_std --target braket                   --emulate %s -o %t && %t | FileCheck %s
 // RUN: nvq++ %cpp_std --target ionq                     --emulate %s -o %t && %t | FileCheck %s
 // RUN: nvq++ %cpp_std --target iqm --iqm-machine Adonis --emulate %s -o %t && %t | FileCheck %s
 // RUN: nvq++ %cpp_std --target oqc                      --emulate %s -o %t && %t | FileCheck %s
 // RUN: nvq++ %cpp_std --target quantinuum               --emulate %s -o %t && %t | FileCheck %s
+// RUN: if $braket_avail; then \
+// RUN:   nvq++ %cpp_std --target braket                 --emulate %s -o %t && %t | FileCheck %s; \
+// RUN: fi
```

Comment:
I don't like all this whitespace and extra, unnecessary formatting. It just makes it hard to cut and paste the command.

Reply:
$braket_avail is replaced by lit, so I don't think you would usually copy-paste the command including that. Hence, I personally find this easier to copy-paste and more readable. That said, I am happy to change it to be one line.